src/Libraries/RevitNodes/Elements/Element.cs

```diff
@@ -154,14 +154,34 @@ public string UniqueId
         /// <summary>
         /// Get the Element Pinned status
         /// </summary>
-        public bool? IsPinned
+        public bool IsPinned
         {
             get 
-            { 
-                return this.InternalElement?.Pinned; 
+            {
+                if (InternalElement == null)
+                    return false;
+                return this.InternalElement.Pinned; 
             }
         }
 
+        /// <summary>
+        /// Checks if two elements are joined
+        /// </summary>
+        /// <param name="otherElement">Element to check</param>
```

Comment:
maybe `Second element to check` ? need to differentiate it from the main `Element` input that will automatically be added to node.